clang/find_clang.cr

```diff
@@ -106,12 +106,12 @@ def shell_split(line)
 end
 
 # Shell split the strings.  Remove first of each, as this is the program name.
-cppflags = shell_split(raw_cppflags)[1..-1]
-ldflags = shell_split(raw_ldflags)[1..-1]
+cppflags = shell_split(raw_cppflags)[1..-1] + shell_split(ENV.fetch("CPPFLAGS", ""))
+ldflags = shell_split(raw_ldflags)[1..-1] + shell_split(ENV.fetch("LDFLAGS"], ""))
```

Comment:
`("LDFLAGS"]`

Reply:
Crap. nice catch.

Reply:
Tried to merge changes without all the formatting